int64 X
int64 Y
---
bool return_value
string[] Label